lib/CL/devices/topology/pocl_topology.c

```diff
@@ -279,9 +279,9 @@ pocl_topology_detect_device_info (cl_device_id device)
     {
       assert (content);
       assert (filesize > 0);
-      unsigned long start, end;
+      unsigned long start = 0, end = 0;
       int items = sscanf (content, "%lu-%lu", &start, &end);
-      assert (items == 2);
+      assert (items == 2 || (items == 1 && start == 0));
```

Comment:
This patch is unclear and seems it might call for trouble. Can make a special case for the RISC-V output (detect with 'hart' or such) instead of this type of magic-like solution?

Reply:
Yeah I know I wrote worthless shit. Just forget it.